package.json

```diff
@@ -5,7 +5,7 @@
   "main": "dist/commonjs/axios.js",
   "scripts": {
     "pretest": "npm run build",
-    "test": "npm run test:karma && npm run test:unit && npm run lint & bundlesize",
```

Comment:
@Khaledgarbaya @axe312ger that's why the tests "didn't run" aka were run in the background.

Reply:
🤦‍♂️

Reply:
:drum: